api/src/main/java/org/openmrs/module/reporting/cohort/definition/library/BuiltInCohortDefinitionLibrary.java

```diff
@@ -125,4 +127,12 @@ public CohortDefinition getDiedDuringPeriod() {
         cd.addParameter(new Parameter("diedOnOrBefore", "reporting.parameter.endDate", Date.class));
         return new MappedParametersCohortDefinition(cd, "diedOnOrAfter", "startDate", "diedOnOrBefore", "endDate");
     }
+
+    @DocumentedDefinition("patientWithAttribute")
```

Comment:
I was thinking since it's used for the /adhocquery resource that it only applies to patients. Should I change it to `personWithAttribute`